### ISSUE TITLE
Code issue

### ISSUE THREAD

**Issue by zkyntu** [crystal250/GSENet#4]
Hi, thanks for your work. I have a question about the angle loss. CLRNet uses L1 loss to regress the angle, which is also used in your [code](https://github.com/crystal250/GSENet/blob/ec1b678492b675824766e74ea6ab0aff6d18f1db/clrnet/models/heads/clr_head.py#L444-L446). Do you test angle loss without L1 loss between the predicted angle and GT? Besides, I found an error in your code: [L451](https://github.com/crystal250/GSENet/blob/ec1b678492b675824766e74ea6ab0aff6d18f1db/clrnet/models/heads/clr_head.py#L451) in clr_head.py: ` angle_loss += a_loss(reg_pred, reg_targets, self.img_h)`?

I try to reproduce your work. Do you have any plans to release the pretrained weights?

Best

**Comment by crystal250** [comment]
We have uploaded the ResNet-101 and DLA-34 models trained on the CULane dataset (.pth files) to help you reproduce our results.
